aten/src/ATen/native/Copy.cpp

```diff
@@ -206,8 +206,11 @@ static Tensor & copy_impl(Tensor & self, const Tensor & src, bool non_blocking)
     return self;
   }
 
+  // Copies from meta are also OK and can be done directly.
+  // TODO: is this right?!
```

Comment:
IIUC, copy should be never called on meta tensor. If the code path ends here, something needs to be fixed upstream.